Moonshot/baddies/TurningHack.gd

```diff
@@ -0,0 +1,9 @@
+extends RayCast2D
+class_name TurningHack
+
+var is_turning: bool = false
+
+func async_change_direction() -> void:
+	is_turning = true
+	yield(get_tree().create_timer(0.2), "timeout")
```

Comment:
pffft, it works 🤷🏻‍♂️

Reply:
Not difficult to read i don't think so no sleep easy ;)